corehq/apps/change_feed/topics.py

```diff
@@ -6,6 +6,7 @@
 META = 'meta'
 
 # new models
+CASE_SQL = 'case-sql'
```

Comment:
topics are a kafka concept - which "feed" you are reading from/writing to. data sources are an HQ concept - given this change how do I get the original document out?

Longer term I had imagined that both couch and sql and push to the same kafka topic and we just have one HQ consumer (pillow) that's about to figure out based on the data source how to get the thing and then send it upstream to elastic, but this seemed better for now since the pillowtop architecture isn't quite there yet